airflow/providers/yandex/CHANGELOG.rst

```diff
@@ -27,6 +27,15 @@
 Changelog
 ---------
 
+3.8.0
+.....
+
+Features
+~~~~~~~~
+
+* ``add Yandex Lockbox secrets backend``
+* ``add support for metadata service for Yandex Cloud connections``
+
```

Comment:
You don't have to add this lines here, this would be autogenerated during new release.